Bigtable/src/V2/BigtableCellBuffer.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Google\Cloud\Bigtable\V2;
+
+use Google\Cloud\Bigtable\V2\Exception\InvalidChunkException;
+
```

Comment:
Would be great to add a PHPDoc here and/or __construct() to let readers know what this class is about.